Dockerfile.ubuntu

```diff
@@ -0,0 +1,7 @@
+FROM python:3.10.6-slim
+WORKDIR /app
+RUN apt update
+RUN apt install -y wget git python3 python3-venv libgl1 libglib2.0-0
+RUN wget -q https://raw.githubusercontent.com/AUTOMATIC1111/stable-diffusion-webui/master/webui.sh
+COPY ./webui.sh ./webui.sh
+CMD ["bash"]
```

Comment:
CI does
```
python launch.py --skip-torch-cuda-test --exit
```
to install dependencies, then quit.

That would of course mean you'd need to `COPY` in the rest of the code (which is prudent anyway).

Reply:
Makes sense, thanks for the tip - I changed it to `COPY` the whole repo directory